sql/updates/world/2024_01_09_Miss_Danna_and_children.sql

```diff
@@ -0,0 +1,487 @@
+DELETE FROM `creature_text` WHERE Entry= 3513;
```

Comment:
Miss Danna and children events is changed maybe legion.I will check behavior in retail server for you.

Reply:
ok, i l waiting